packages/os/kdump-tmpfiles.conf

```diff
@@ -0,0 +1,4 @@
+d /local/boot 0700 root root -
+d /local/var/log/kdump 0700 root root -
+T /local/boot - - - - security.selinux=system_u:object_r:secret_t:s0
```

Comment:
Can we just mount this at `/boot`? Putting it under `/local` seems weird.

Also the label should be `os_t` since it's not a secret.